src/Useful.ts

```diff
@@ -10,6 +10,16 @@ export function rgbToHex(s: string): string {
   return '#' + ((1 << 24) + (rgb[0] << 16) + (rgb[1] << 8) + rgb[2]).toString(16).toUpperCase().slice(1);
 }
 
+/**
+ * Tests if a given string is a valid hex number
+ *
+ * @param {string} hex The string to test
+ * @returns {boolean} Returns true or false as the result of testing the string
+ */
+export function isHex(hex: string): boolean {
+  return (hex.indexOf('#') === 0);
```

Comment:
Or use e.g. https://www.npmjs.com/package/is-hexcolor??? Totally unsure might also be way over-architected.

Reply:
Possible, yes. It would be safer. I assume that MapServer exports a valid MapFile and that valid hex values (with #) are specified there.  
But if these files were edited manually afterwards, such a check is of course safer. 

This function is used to check before an already existing conversion of color values, whether they are already hex values, which then do not have to be converted any more. Wrong values would of course still be converted incorrectly afterwards.

Reply:
I would also prefer not to have an extra dependency just for a single check. especially if the complete code of that dependency is as follows:

```
'use strict'

var hexColorRegex = require('hex-color-regex');

module.exports = function isHexcolor (hex) {
  return hexColorRegex({strict: true}).test(hex)
}
```

[source](https://github.com/tunnckoCore/is-hexcolor/blob/master/index.js)

Reply:
The test then remains as a simple check as currently submitted.

Reply:
Fine for me.